cmd/otelcontribcol/builder-config.yaml

```diff
@@ -257,3 +257,7 @@ providers:
 # When using `make genotelcontribcol`, a `replaces` section is appended to this
 # file before passing it to OCB, to ensure that local versions are used for all
 # Contrib modules.
+replaces:
+  - go.opentelemetry.io/collector/extension/extensionauth => go.opentelemetry.io/collector/extension/extensionauth v0.0.0-20250226024140-8099e51f9a77
+  - go.opentelemetry.io/collector/extension/extensionauth/extensionauthtest => go.opentelemetry.io/collector/extension/extensionauth/extensionauthtest v0.0.0-20250226024140-8099e51f9a77
+  - go.opentelemetry.io/collector/service/hostcapabilities => go.opentelemetry.io/collector/service/hostcapabilities v0.0.0-20250226024140-8099e51f9a77
```

Comment:
cc @mx-psi as the next releaser. These replaces need to be removed once the new modules are released. They are referenced by v0.0.0-00010101000000-000000000000 and v0.120.0 which are invalid. So I've put replaces here instead of trying to fix it in core which would require a few commits pointing to versions from parent commits